### PR TITLE
Add public subscription logic

### PR DESCRIPTION
**Pull request by prokhorind**

### CODE REVIEW

File: src/main/java/com/mattermost/integration/figma/api/figma/file/service/FigmaFileServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Component
```
Comment (OstapMelnychuk) [inline src/main/java/com/mattermost/integration/figma/api/figma/file/service/FigmaFileServiceImpl.java:15]: Why are you not using service annotation?

File: src/main/java/com/mattermost/integration/figma/input/mm/form/Field.java [hunk 2]
```diff
@@ -2,8 +2,10 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
+import lombok.experimental.SuperBuilder;
 
 @Data
+@SuperBuilder
```
Comment (OstapMelnychuk) [inline src/main/java/com/mattermost/integration/figma/input/mm/form/Field.java:8]: I think Builder annotation should be here

File: src/main/java/com/mattermost/integration/figma/api/figma/file/service/FigmaFileServiceImpl.java (hunk 1 above)
Comment (prokhorind) [reply src/main/java/com/mattermost/integration/figma/api/figma/file/service/FigmaFileServiceImpl.java]: fixed

File: src/main/java/com/mattermost/integration/figma/input/mm/form/Form.java [hunk 3]
```diff
@@ -1,10 +1,14 @@
 package com.mattermost.integration.figma.input.mm.form;
 
 import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
 
 import java.util.List;
 
 @Data
+@SuperBuilder
```
Comment (OstapMelnychuk) [inline src/main/java/com/mattermost/integration/figma/input/mm/form/Form.java:10]: I think Builder annotation should be here

**Review by OstapMelnychuk — APPROVED** [review]

**Pull request merged**